src/mds/CInode.cc

```diff
@@ -1037,12 +1039,12 @@ void CInode::fetch(MDSInternalContextBase *fin)
   object_t oid = CInode::get_object_name(ino(), frag_t(), "");
   object_locator_t oloc(mdcache->mds->mdsmap->get_metadata_pool());
 
+  // Old on-disk format: inode stored in xattr of a dirfrag
```

Comment:
The thought crossed my mind, but there was no functional need to do it for the ENOENT case.  We can easily remove it in a separate patch though.